Resources/Prototypes/Recipes/Reactions/chemicals.yml

```diff
@@ -57,6 +71,94 @@
       scaled: true #Scaled proportionally to amount of potassium and water
       maxScale: 30 #Explosion strength stops scaling at 30 potassium + 30 water
 
+- type: reaction
+  id: react.Smoke
+  reactants:
+    chem.Phosphorus:
+      amount: 1
+    chem.Potassium:
+      amount: 1
+    chem.Sugar:
+      amount: 1
+  effects:
+    - !type:AreaReactionEffect
+      rangeConstant: 0
+      rangeMultiplier: 1.1 #Range formula: rangeConstant + rangeMultiplier*sqrt(ReactionUnits)
+      maxRange: 10
+      duration: 10
+      spreadDelay: 0.5
+      removeDelay: 0.5
+      diluteReagents: false
+      prototypeId: Smoke
+      sound: /Audio/Effects/smoke.ogg #Taken from https://github.com/tgstation/tgstation/blob/master/sound/effects/smoke.ogg
```

Comment:
Dont put licenses inside the prototype, there should be an license.txt inside the sound folder, put it in there